### PR TITLE
#35 - Update docs

### PR DESCRIPTION
**Pull request by Jeckerson**
Related PR phalcon/docs#2652

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,38 +3,78 @@
 [![Discord](https://img.shields.io/discord/310910488152375297?label=Discord)](http://phalcon.link/discord)
 [![codecov](https://codecov.io/gh/phalcon/migrations/branch/master/graph/badge.svg)](https://codecov.io/gh/phalcon/migrations)
 
-Generate or migrate database changes via migrations.
+Generate or migrate database changes via migrations.  
+Main idea of Phalcon migrations is to automatically detect changes and morphing without writing manual migrations.
+
+## Full documentation
+
+https://docs.phalcon.io/4.0/en/db-migrations
```
Comment (ruudboon) [inline README.md]: Can we link to https://docs.phalcon.io/latest/en/db-migrations this will be the default after Seo changes.

File: README.md [hunk 2]
```diff
@@ -3,38 +3,78 @@
...
 
 ## Requirements
 
 * PHP >= 7.2
 * Phalcon >= 4.0.0
-* PHP ext-posix
```
Comment (sergeyklay) [inline README.md]: Are POSIX functions deleted from the code?

File: README.md [hunk 3]
```diff
@@ -3,38 +3,78 @@
...
 ```
 
-## Usage
-
+```json
+{
+    "require": {
+        "phalcon/migrations": "^1.1"
+    }
+}
```
Comment (sergeyklay) [inline README.md]: Why we need this in the readme? Is composer require / install not enough or not completely clear?

File: README.md (hunk 2 above)
Comment (Jeckerson) [reply README.md]: Nope, but only for Linux

**Pull request merged**